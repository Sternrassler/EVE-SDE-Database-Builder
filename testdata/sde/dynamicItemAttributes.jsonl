{"typeID":34,"attributeID":4,"mutatorID":null,"playerID":null}
{"typeID":35,"attributeID":4,"mutatorID":null,"playerID":null}
{"typeID":36,"attributeID":37,"mutatorID":1,"playerID":12345}
