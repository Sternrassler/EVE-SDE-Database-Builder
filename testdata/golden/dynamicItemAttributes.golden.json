[
  {
    "typeID": 34,
    "attributeID": 4
  },
  {
    "typeID": 35,
    "attributeID": 4
  },
  {
    "typeID": 36,
    "attributeID": 37
  }
]